scripts/zones/RoMaeve/mobs/Shikigami_Weapon.lua

```diff
@@ -49,6 +49,7 @@ entity.onMobSpawn = function(mob)
 end
 
 entity.onMobEngaged = function(mob, target)
+    mob:setStatus(xi.status.NORMAL)
     mob:setStatus(xi.status.UPDATE)
```

Comment:
Double setStatus for this shouldn't be needed and indicates something happening incorrectly core side if this affects anything at all.

Reply:
![Teo_2023 09 06_180904](https://github.com/LandSandBoat/server/assets/6871475/f72cc2fd-819a-46fe-aab7-9edb40dbee07)

So I am not seeing the bug on stock LSB

Reply:
Is he able to be engaged?  I think that's what the original bug report to us said.  I'll retest to see if that's the case.

Reply:
we engaged and killed it. my Char was the bystanding observer.

Reply:
I can say, that I have been able to cast around it and was able to kill it with each pop.

Reply:
Thank you, I will close the PR.

Reply:
This may be downstream specific